src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public Figure getRandomFigure() {
+        Random figureNumber = new Random();
+        Random figurePart = new Random();
```

Comment:
you can use the same random for different purposes - let's create only one variable